ocdskingfisherarchive/collection.py

```diff
@@ -0,0 +1,45 @@
+import json
+import os
+import tempfile
+
+
+class Collection:
+
+    def __init__(self, config, database_id, source_id, data_version):
+        self.config = config
+        self.database_id = database_id
+        self.source_id = source_id
+        self.data_version = data_version
+
+    def write_meta_data_file(self):
+        data = {
+            'database_id': self.database_id,
+        }
+        file_descriptor, filename = tempfile.mkstemp(prefix='archive', suffix='.json')
+        with open(filename, 'w') as file:
+            json.dump(data, file, indent=4)
+        os.close(file_descriptor)
+        return filename
+
+    def write_data_file(self):
+        file_descriptor, filename = tempfile.mkstemp(prefix='archive', suffix='.tar')
+        os.close(file_descriptor)
+
+        dir_to_add = self.config.directory_data + '/' + self.source_id + '/' + \
```

Comment:
I'm not crazy about passing around a config object. Instead, just pass the data directory to `__init__` (and any others that are specifically needed in future).

Reply:
There are going to be many different config vars by the time we've finished tho, so it seemed easier to read/edit/work with as an object. Can we leave it at this for now and review at end?

Reply:
Also, passing an object makes it easy to work out which config vars a function needs, which might otherwise be difficult. DataBaseProcess at first glance only needs DB connection params, but actually as it makes Collections objects it needs more than just that. It would be nice to avoid these problems for now while I work on the main algorithm.

Reply:
Sure, we can change the method signatures later. I created https://github.com/open-contracting/kingfisher-archive/issues/20